code/modules/recycling/conveyor2.dm

```diff
@@ -116,6 +118,7 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 /obj/machinery/conveyor/proc/update()
 	if(stat & BROKEN || !operable || stat & NOPOWER)
 		operating = FALSE
+		update_icon()
```

Comment:
this should probably be called either way tbh

Reply:
It's already called where the value changes and doing it here stops it being called twice

Reply:
to think you want to call this twice but haven't called your mother once